### PR TITLE
Added operation config choice for each condition

### PR DESCRIPTION
**Pull request by BenMatase**
Support or and and specification for the operation to combine the keys together for a condition

### CODE REVIEW

File: plugins/processors/t128_filter/README.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ The `t128_filter` filters out metrics passing through it. This processor is usef
     ##  * "regex": go flavored regex comparison
```
Comment (BenMatase) [inline plugins/processors/t128_filter/README.md:18]: I'm not going to do this rn, but I feel like most of this readme could be dynamically filled in by the code. Like this sample configuration is exactly the same as the telegraf sample configuration. The examples could be derived from testcases. Then nothing would get out of sync and wouldn't have to keep two places up to date.

Comment (gregschrock) [reply plugins/processors/t128_filter/README.md:18]: sounds like you're describing [rust](https://doc.rust-lang.org/rustdoc/documentation-tests.html) 🙂

**Review by gregschrock — APPROVED** [review]
So clean!

File: plugins/processors/t128_filter/README.md (hunk 1 above)
Comment (BenMatase) [reply plugins/processors/t128_filter/README.md:18]: Oh man, that looks sweet

File: plugins/processors/t128_filter/README.md [hunk 2]
```diff
@@ -18,6 +18,12 @@ The `t128_filter` filters out metrics passing through it. This processor is usef
...
     # mode = "exact"
 
+    ## Operation dictates how to combine the condition's tag matching
+    ## Valid values are:
+    ##  * "and": logical and the results together
```
Comment (agrawalkaushik) [inline plugins/processors/t128_filter/README.md:23]: why the `*` in front of the `"and"` and `"or"`?

**Review by agrawalkaushik — APPROVED** [review]

File: plugins/processors/t128_filter/README.md (hunk 2 above)
Comment (BenMatase) [reply plugins/processors/t128_filter/README.md:23]: I was trying to make it a bulleted list

**Pull request merged**